### PR TITLE
Fix #535,Fix part of #530: Material card-view issue fix

### LINKED ISSUE #535: material card corner white spacing.
At the corners of the card you would find white spacing on both top corners, and white card corner below the stroke corner borders of bottom rounded corners. https://github.com/oppia/oppia-android/pull/323#discussion_r344704934

### PR DESCRIPTION
**Pull request by nikitamarysolomanpvt**
## Explanation
To use Material Components, we need to either use a Theme.MaterialComponents.* theme or a Theme.MaterialComponents.*.Bridge theme.

See here for more info on migrating to a MaterialComponents theme (recommended) - https://github.com/material-components/material-components-android/blob/master/docs/getting-started.md#4-change-your-app-theme-to-inherit-from-a-material-components-theme

Additionally, we don't need to add two dependencies for Material in your :app module's build.gradle file. Below are the latest stable, beta and alpha versions of the library. Should use a single version, depending on the features vs. stability our app needs.
So i removed com.google.android.material:material:1.0.0 and used alpha.

stable: com.google.android.material:material:1.0.0
beta: com.google.android.material:material:1.1.0-beta02
alpha: com.google.android.material:material:1.2.0-alpha02
Card corner white spacing fixed for all API

### **Oppia Android API 19 4.4.2 (Kitkat) UI issues.**

**Chapter List (Mathew goes..)**

Untitled

- [x] Material Cards UI issues 

- [x] Drop shadow

- [x] Card borders

**Review** 

Untitled

- [x] Material Cards UI issues 

- [x] Drop shadow

- [x] Card borders

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by rt4914 — APPROVED** [review]
LGTM. My only concern is regarding the alpha version. @BenHenning will need to finalise this whether we can use alpha version for this library or not.

**Review by veena14cs — APPROVED** [review]
LGTM

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
-  <style name="OppiaTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+  <style name="OppiaTheme" parent="Theme.MaterialComponents.Light.DarkActionBar">
     <item name="colorPrimary">@color/colorPrimary</item>
     <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
     <item name="colorAccent">@color/colorPrimary</item>
   </style>
-
-  <style name="OppiaThemeWithoutActionBar" parent="Theme.AppCompat.Light.NoActionBar">
+  name="Theme.MyApp" parent="Theme.MaterialComponents.DayNight"
```
Comment (BenHenning) [inline app/src/main/res/values/styles.xml]: This appears to be a typo--remove? Or at least, it's not part of a tag so it seems unintentional.

Comment (nikitamarysolomanpvt) [reply app/src/main/res/values/styles.xml]: Removed

**Comment by nikitamarysolomanpvt** [comment]
Duplicate (approach 1) of #537.

**Pull request closed without merging**